pom.xml

```diff
@@ -660,7 +660,7 @@
 				<plugin>
 					<groupId>org.codehaus.mojo</groupId>
 					<artifactId>build-helper-maven-plugin</artifactId>
-					<version>3.5.0</version>
```

Comment:
The changes in this file do not look related to the ticket.

Reply:
I will clean this up. Some changes were made automatically.

Reply:
Reverted